core/src/tangram.cpp

```diff
@@ -63,19 +62,34 @@ void initialize(const char* _scenePath) {
         logMsg("Loading Tangram scene file: %s\n", sceneRelPath.c_str());
         auto sceneString = stringFromResource(sceneRelPath.c_str());
 
-        SceneLoader loader;
+        if (SceneLoader::loadScene(sceneString, *m_scene)) {
+            m_tileManager->setScene(m_scene);
 
-        loader.loadScene(sceneString, *m_scene, *m_tileManager, *m_view);
+            glm::dvec2 projPos = m_view->getMapProjection().LonLatToMeters(m_scene->startPosition);
+            m_view->setPosition(projPos.x, projPos.y);
+            m_view->setZoom(m_scene->startZoom);
+        }
 
         m_skybox = std::unique_ptr<Skybox>(new Skybox("img/cubemap.png"));
         m_skybox->init();
-
     }
 
     logMsg("finish initialize\n");
 
 }
 
+void loadScene(const char* _scenePath) {
+
+    m_scene = std::make_shared<Scene>();
+
+    logMsg("Loading Tangram scene file: %s\n", _scenePath);
+    auto sceneString = stringFromResource(_scenePath);
```

Comment:
Before loading resources for a new scene, the resource root location needs to be set, as here: https://github.com/tangrams/tangram-es/blob/master/core/src/tangram.cpp#L40